### DISCUSSION TITLE
Please add the radar

### DISCUSSION THREAD

**Discussion by IamcomingfromJUPITER** [MartinTheDragon/Nuclear-Tech-Mod-Remake#19] [Ideas]
Hi! can you add the radar tha
![download](https://user-images.githubusercontent.com/113767946/190850914-aadd4bd3-1731-431a-9bef-8fae065635f3.jpg)
t included in the original one? (For detecting missiles)

**Comment by MartinTheDragon** [comment]
As soon as I continue missiles, yes, of course

**Reply by IamcomingfromJUPITER** [reply]
okay thanks
